rest/dbHandler.go

```diff
@@ -127,7 +114,26 @@ func (dbHandler DbHandler) deleteTrackPage(w http.ResponseWriter, r *http.Reques
 		return
 	}
 
-	// Send response
-	w.WriteHeader(c)
+	w = prepareGeneralResponse(w, []byte(trackID), constant.TextPlain, nil, "deleteTrackPage")
+	return
+}
+
+// Prepares a general response to send back to the client, setting various common variables in the ResponseWriter
+func prepareGeneralResponse(unPrepW http.ResponseWriter, rawData []byte, contentType string, jsonError error, callerFunc string) (prepW http.ResponseWriter) {
```

Comment:
Why does this method take in an error? Think you only call it with nil previously, and if an error is to be sent back to the user, why sent the whole error stack when you only use err.Error? e.i. the string version. If you manage to get rid of this param you will also drop down to the allowed param count

Reply:
The reason I do this is to cut down on the repeated code basically when it comes to error handling. If I remove that there we'll need that error handling code to basically be there in each of the 4 functions instead. It's used when we parse json in getTracks and insertTrack, as the parsing can give an error and is then handled by the response code. I've moved the error code back up, we may make it a new task to improve the look of the current error logging, the current way of logging messages feels rather clunky.